lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -7,38 +7,43 @@ module FlashHelper
     #   <a href="" class="close">&times;</a>
     # </div>
     DEFAULT_KEY_MATCHING = {
-      :alert     => :alert,
-      :notice    => :success,
-      :info      => :info,
-      :secondary => :secondary,
-      :success   => :success,
-      :error     => :alert,
-      :warning   => :warning
+      :alert     => 'alert',
+      :notice    => 'success',
+      :info      => 'info',
+      :secondary => 'secondary',
+      :success   => 'success',
+      :error     => 'alert',
+      :warning   => 'warning'
     }
-    def display_flash_messages(key_matching = {})
+
+    def display_flash_messages(opts={})
+      key_matching = opts.delete(:key_matching) || {}
       key_matching = DEFAULT_KEY_MATCHING.merge(key_matching)
-      key_matching.default = :standard
+
+      dismiss_toggle = opts[:dismiss_toggle]
+      alert_style = opts[:style]
 
       capture do
         flash.each do |key, value|
           next if FoundationRailsHelper.configuration.ignored_flash_keys.include? key.to_sym
           alert_class = key_matching[key.to_sym]
```

Comment:
@dsandstrom this is the only place the hash is used. I can't see any reason that they should be symbols